source/geometry_model/ellipsoidal_chunk.cc

```diff
@@ -110,6 +113,40 @@ namespace aspect
       return phi_theta_d;
     }
 
+    template <int dim>
+    Point<3>
+    EllipsoidalChunk<dim>::EllipsoidalChunkGeometry::push_forward_topography(const Point<3> &phi_theta_d_hat) const
+    {
+      const double d_hat = phi_theta_d_hat[2]; // long, lat, depth
+      Point<dim-1> phi_theta;
+      const double rad_to_degree = 180/numbers::PI;
+      if (dim == 3)
```

Comment:
It seems this whole geometry only works for 3D models? If so please add another Assertion here. I know that there are some others already, but somebody might copy this function to a new material model and then wonder about the missing initialization for 2D.